praw/models/reddit/live.py

```diff
@@ -1,14 +1,14 @@
 """Provide the LiveThread class."""
 from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Optional, Union
 
-from ...const import API_PATH
-from ...util.cache import cachedproperty
-from ..list.redditor import RedditorList
-from ..listing.generator import ListingGenerator
-from ..util import stream_generator
 from .base import RedditBase
 from .mixins import FullnameMixin
 from .redditor import Redditor
+from ..list.redditor import RedditorList
+from ..listing.generator import ListingGenerator
+from ..util import stream_generator
+from ...const import API_PATH
+from ...util.cache import cachedproperty
```

Comment:
Why did these imports switch order?

Reply:
> Why did these imports switch order?

Hm, this wasn't something I intentionally did, my IDE auto sorts imports when I hit the shortcut for reformat and I must have hit it habitually. Though, except for a few edge cases, I could sort the imports for all files to be consistent.

Reply:
I rely on a tool called `isort`to deal with import order.